### ISSUE TITLE
RedisFeatureStore does not offer ssl_ca_certs kwarg

### ISSUE THREAD

**Issue by cn-alexboyd** [launchdarkly/python-server-sdk#159]
**Is your feature request related to a problem? Please describe.**

I would love for the RedisFeatureStore to allow for me to specify a host, port, password, SSL, and ssl_ca_cert if applicable when trying to connect to a Redis cluster that has TLS/SSL enabled.

**Describe the solution you'd like**

Ideally, I could use the existing `RedisFeatureStore.new_feature_store()` method with a new set of kwargs instead of just the connection url. 

```python
store = Redis.new_feature_store(host=..., port=..., password=..., ssl=True, ssl_ca_certs=...)
```

**Describe alternatives you've considered**

The required [redis](https://redis-py.readthedocs.io/en/stable/#redis.Redis) package already supports these args, I think the adjustment will be using something other than `redis.ConnectionPool.from_url` during initialization of _RedisFeatureStoreCore.

**Comment by danie1k** [comment]
SSL-related and other Redis connection options can be easily set directly in the URL, for example:

```python
qs = urllib.parse.urlencode({
    "ssl_cert_reqs": "required",
    "ssl_ca_certs": "/absolute/path/to/ca.crt",
    "ssl_keyfile": "/absolute/path/to/client.key",
    "ssl_certfile": "/absolute/path/to/client.crt",
})

store = Redis.new_feature_store(f'rediss://:password@host:port/0?{qs}')
```

**Comment by keelerm84** [comment]
Fixed in 7.5.0
